release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -172,6 +211,17 @@ the web console] for more information.
 
 You can now open a Red Hat Support case from the help menu in the web console.
 
+[id="ocp-4-3-developer-perspective"]
+==== Developer Perspective
+
+You can now use the Developer Perspective to:
```

Comment:
s/Perspective/perspective
See: https://docs.openshift.com/container-platform/4.2/web-console/odc-about-developer-perspective.html (ignoring the h2 that mysteriously capitalizes perspective)